src/frontend/screens/Game/GamePicture/index.tsx

```diff
@@ -6,10 +6,17 @@ import fallbackImage from 'frontend/assets/heroic_card.jpg'
 
 interface Props extends React.ImgHTMLAttributes<HTMLImageElement> {
   art_square: string
+  logo?: string | undefined
```

Comment:
do we need the `| undefined`? I think because it's optional with the `?` it already includes undefined as a possibility